### PR TITLE
Twilio Credentials

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Rename Twilio API Key to Twilio Credentials and add more patterns for the rule

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark

### CODE REVIEW

File: credsweeper/rules/config.yaml [hunk 1]
```diff
@@ -644,15 +644,39 @@
   target:
     - code
 
-- name: Twilio API Key
-  severity: high
+- name: Twilio Credentials
+  severity: critical
```
Comment (babenek) [inline credsweeper/rules/config.yaml:648]:
```suggestion
  severity: high
```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -374,7 +377,8 @@ jobs:
       - name: Checkout CredData
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          repository: Samsung/CredData
+          repository: babenek/CredData
+          ref: twillio
```
Comment (babenek) [inline .github/workflows/benchmark.yml:381]:
```suggestion
          repository: Samsung/CredData
          ref: main
```

File: .github/workflows/benchmark.yml [hunk 3]
```diff
@@ -187,7 +189,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:193]:
```suggestion
          repository: Samsung/CredData
          ref: main
```

File: .github/workflows/benchmark.yml [hunk 4]
```diff
@@ -85,7 +86,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:90]:
```suggestion
          repository: Samsung/CredData
          ref: main
```

File: .github/workflows/benchmark.yml [hunk 5]
```diff
@@ -30,7 +30,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:34]:
```suggestion
          repository: Samsung/CredData
          ref: main
```

**Review by babenek — COMMENTED** [review]
TODO: rollback after https://github.com/Samsung/CredData/pull/173 merge

**Review by kmnls — APPROVED** [review]
Agree

**Review by iuriimet — APPROVED** [review]

**Pull request merged**